libraries/libfc/include/fc/crypto/bls_private_key.hpp

```diff
@@ -6,8 +6,7 @@
 namespace fc::crypto::blslib {
 
    namespace config {
-      constexpr const char* bls_private_key_base_prefix = "PVT";
-      constexpr const char* bls_private_key_prefix = "BLS";
+      constexpr std::string_view bls_private_key_prefix = "PVT_BLS_";

```

Comment:
Why `std::string_view` and not `std::string`? With `std::string` you won't have to construct a new `std::string` every time the `to_string()` function is called.